src/Core/View.js

```diff
@@ -340,6 +386,21 @@ View.prototype.getLayers = function getLayers(filter) {
     return result;
 };
 
+/**
+ * Gets the layer by identifier.
+ *
+ * @param {String}  layerId  The layer identifier
+ * @return {Layer}  The layer by identifier.
+ */
+
+View.prototype.getLayerById = function getLayers(layerId) {
+    const layers = this.getLayers(l => l.id === layerId);
+    if (layers.length === 1) {
```

Comment:
Unnecessary ? It should be impossible to have two layers with the same id.

Reply:
done